jarvis.py

```diff
@@ -2449,6 +2452,26 @@ def set_brightness(level):
         os.system("""osascript -e 'tell application "System Events"' -e 'key code 145' -e ' end tell'""")
     for _ in range(level):
         os.system("""osascript -e 'tell application "System Events"' -e 'key code 144' -e ' end tell'""")
+                                            
+                                          
+def battery():
+    """
+    Provides basic info about battery info win32
+    """
+    if operating_system == 'Windows':
+        # https://stackoverflow.com/a/41988506/6771356
+
+        def secs2hours(secs):
+            mm, ss = divmod(secs, 60)
+            hh, mm = divmod(mm, 60)
+            return "%d:%02d:%02d" % (hh, mm, ss)
+        batt = psutil.sensors_battery()
```

Comment:
You import `sensors_battery()` from psutil but you are using `psutil.sensors_battery()` which will break the code as Jarvis doesn't know what psutil is.